include/configs/mx6ullevk_siklu.h

```diff
@@ -40,7 +40,21 @@
 #define CONFIG_CMD_MTDPARTS	/* Enable MTD parts commands */
 #define CONFIG_MTD_DEVICE	/* needed for mtdparts commands */
 #define MTDIDS_DEFAULT		"nand0=gpmi-nand" //      SPI  ;spi1=spi_flash
-#define MTDPARTS_DEFAULT   "mtdparts=gpmi-nand:128k(env),128k(env2),128k(env_t),40M(uimage0),"  \
+#define CONFIG_SYS_MTDPARTS_RUNTIME
+
+/* Note for unified U-Boot version:
+ * MTDPARTS_DEFAULT is conditionally checked in cmd/mtdparts.c.
+ * For the unified U-Boot we do not define MTDPARTS_DEFAULT because there is no
+ * single default, it depends on board type. Instead, we set
+ * CONFIG_SYS_MTDPARTS_RUNTIME in configs/mx6ull_14x14_skl_defconfig and we 
```

Comment:
`CONFIG_SYS_MTDPARTS_RUNTIME` is actually defined a few lines above here.

Reply:
Fixed comment.